contrib/ci/deps.sh

```diff
@@ -155,9 +155,6 @@ if [[ "$DISTRO_BRANCH" == -debian-* ]]; then
         libuid-wrapper
         libpam-wrapper
         python-pytest
-        python-ldap
-        python-ldb
-        python-requests
```

Comment:
Perhaps move this to different commit with explanation?

Reply:
Done.